test/system/505-networking-pasta.bats

```diff
@@ -51,7 +51,7 @@ function pasta_test_do() {
     local bytes="${7}"
 
     # DO NOT ADD ANY CODE ABOVE THIS LINE! Especially skips: that could
-    # lead to the crossreference check not running in CI.
+    # lead to the cross-reference check not running in CI.
```

Comment:
Should you choose to re-push (to fix the "effects" above), could you remove this correction? This code is removed in #19100 so we have a conflict.